packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -1235,6 +1235,66 @@ function commitTransitionProgress(offscreenFiber: Fiber) {
   }
 }
 
+function removeSuspenseBoundariesFromSubtree(finishedWork) {
+  let node: Fiber = finishedWork;
+  while (true) {
+    // if is hidden we don't need to traverse anymore
+    if (node.tag === SuspenseComponent && node.child.memoizedState !== null) {
+      const offscreenInstance: OffscreenInstance = node.child.stateNode;
+      // If the boundary is shown we don't hae to do this
+      if (offscreenInstance.pendingMarkers !== null) {
+        offscreenInstance.pendingMarkers.forEach(markerInstance => {
+          if (markerInstance.pendingBoundaries.has(offscreenInstance)) {
+            markerInstance.pendingBoundaries.delete(offscreenInstance);
+
+            if (markerInstance.transitions !== null) {
+              if (markerInstance.name) {
+                addMarkerProgressCallbackToPendingTransition(
+                  markerInstance.name,
+                  markerInstance.transitions,
+                  markerInstance.pendingBoundaries,
+                );
+              } else {
+                markerInstance.transitions.forEach(transition => {
+                  addTransitionProgressCallbackToPendingTransition(
+                    transition,
+                    markerInstance.pendingBoundaries,
+                  );
+                });
+              }
+            }
+          }
+        });
+      }
+    } else if (
+      (node.tag === OffscreenComponent || node.tag === LegacyHiddenComponent) &&
+      (node.memoizedState: OffscreenState) !== null &&
+      node !== finishedWork
+    ) {
+      // Found a nested Offscreen component that is hidden.
+      // Don't search any deeper. This tree should remain hidden
+    } else if (node.child !== null) {
```

Comment:
Is there a flag that we can check to know if there's a suspended suspense boundary in here?